alerts/geomodel_alert.json

```diff
@@ -0,0 +1,25 @@
+{
+  "elasticsearch_address": "",
+  "localities": {
+    "es_index": "localities",
+    "valid_duration_days": 1,
+    "radius_kilometres": 50
+  },
+  "events": [
+    {
+      "es_index": "events-*",
+      "search_window": {
+        "minutes": 5
+      },
+      "lucene_query": "tags:auth0",
+      "username_path": "details.username"
+    }
+  ],
+  "alerts": {
+    "es_index": "alerts-*",
+    "whitelist": {
```

Comment:
I think it's a bit weird to have the whitelist be under the alerts key, what if we made `whitelist` be a key under root?